src/id.rs

```diff
@@ -202,6 +205,9 @@ impl EventId {
     /// The report destination buffer is too small to fit a header and/or the frontier clocks
     pub const EVENT_INSUFFICIENT_REPORT_BUFFER_SIZE: EventId =
         EventId(unsafe { NonZeroU32::new_unchecked(EventId::MAX_INTERNAL_ID - 5) });
+    /// The probe successfully initialized itself. The probe id is stored in the payload.
+    pub const EVENT_PROBE_INITIALIZED: EventId =
```

Comment:
Not sure why we need to store the probe id in the payload here, since recorded events are inherently localized to a probe.